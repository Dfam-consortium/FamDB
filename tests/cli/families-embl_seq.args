families
--format
embl_seq
2
