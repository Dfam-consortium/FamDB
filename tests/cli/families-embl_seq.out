CC   <copyright header>
ID   TEST0003; SV 1; linear; DNA; STD; UNC; 4 BP.
NM   Test family TEST0003
XX
AC   TEST0003;
XX
XX
SQ   Sequence 4 BP; 0 A; 1 C; 2 G; 1 T; 0 other;
     ggtc                                                               4
//
